scripts/generator/graphql_generator/csharp/Editor/ShopifyBuildPipeline.cs.erb

```diff
@@ -0,0 +1,61 @@
+namespace <%= namespace %>.SDK {
```

Comment:
I guess that depends if we are considering bundling the sample app into the `.unitypackage`. 
If we are including a sample app, we would ideally have our own Build pipeline that would configure the Xcode project so that the sample app will run.